models/traffic_metrics/ga4_metrics__aggregate_day_tofu.sql

```diff
@@ -1,10 +1,20 @@
+--TODO: add blended user_id to handle cookie deletes or same users accessing site from different devices
+--TODO: better to use traffic_source_medium or event params medium?
 WITH src AS (
     SELECT *
-    FROM {{ ref('ga4_metrics__sessions_stitched') }}
-),
-
-filtered AS (
-    SELECT *
+    FROM {{ ref('ga4_metrics__sessions') }}
+),
+
+filtered_sessions AS (
+    SELECT *,
+           -- mediums
+           CASE WHEN traffic_source_medium IS NULL OR traffic_source_medium = '(none)' THEN 'organic'
```

Comment:
@macmietek Same as with the cac PR, we have the option to take the medium value from the event parameter or the medium value from the traffic_source_medium field. I am using traffic_source_medium as I think that should be more accurate. Let me know if you think otherwise 🤔

Reply:
`traffic_source_medium` is correct. But `traffic_source_medium = '(none)' ` would typically mean `direct` traffic, not organic. Organic traffic typically goes with `organic` medium, organic traffic name, and google (or other search engine) traffic source.